.github/workflows/WebAutomatedTests.yml

```diff
@@ -3,8 +3,14 @@ name: Run automated end-to-end tests
 on:
     push:
         branches: [master]
+        paths:
+            - packages/tools/**/*
+            - packages/{customWidgets,pluggableWidgets}/*-web/**/*
```

Comment:
Doing this change now we are basically removing the requirement for e2e automated for Atlas..

Reply:
updated